.github/workflows/ci.yml

```diff
@@ -66,9 +66,9 @@ jobs:
         cp -r build/ ./bwc-test/
         mkdir ./bwc-test/src/test/resources/security_plugin_version_no_snapshot
         cp build/distributions/opensearch-security-${security_plugin_version_no_snapshot}.zip ./bwc-test/src/test/resources/${security_plugin_version_no_snapshot}
-        mkdir bwc-test/src/test/resources/2.3.0.0
-        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.3.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-security-2.3.0.0.zip
-        mv opensearch-security-2.3.0.0.zip bwc-test/src/test/resources/2.3.0.0/
+        mkdir bwc-test/src/test/resources/2.4.1.0
```

Comment:
This should remain the immediately preceding minor version. If the goal is to test BWC from 2.4.0.0 -> 2.4.1.0 then it can be set as 2.4.0.0